lib/Master.php

```diff
@@ -202,7 +202,12 @@ protected function redirectUser($uid, $password, $location, array $options = [])
 			]
 		);
 
-		if($isClient) {
+		$requestUri = $this->request->getRequestUri();
+		$isDirectWebDavAccess = strpos($requestUri, 'remote.php/webdav') !== false;
```

Comment:
I'm not 100% certain but some clients might use `remote.php/dav/` as well

Reply:
good point, I will check for this end point as well